docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+  
+```bash
+# avalanche Subnet deploy <SubnetName>
+$ avalanche Subnet deploy game
+
+# choose local network as deployment target
+
+> Metamask connection details (any node URL from above works):
+> RPC URL:          http://127.0.0.1:27200/ext/bc/HGwbjkQsvrqy2mncMWsA2YnrSWEybVmLcZJwz7nb5gNVvcPoi/rpc
+> Funded address:   0x0000000b9af48743ef1188f3F20c9b8B90F52a5b with 1000
+> Network name:     game
+> Chain ID:         321123
+> Currency Symbol:  TEST
+```
+
+Awesomeness! Great work!
+
+# Deploying the Game Contract
+
+## Metamask Configuration
+
+Before going to the next step, make sure you've added the Subnet to metamask by following the same steps in the [`Deploying the Subnet`](#deploying-the-Subnet) section. This time you don't need to import the account as you're already using your own account. When adding the network, I've set the currency symbol to `LUCK`.
+
+![Metamask account look](.github/images/metamask3.png)
+## Metamask Gas Configuration
+
+**! IMPORTANT**
+
+Because we've set the base fee to 60 nAvax, you may need to increase the `Max priority fee` when confirming the transaction on metamask popup. Otherwise the transaction may fail.
+
+Here is how:
+
+![change gas fee](.github/images/changegasfee.gif)
+
+If you're using the *Enhanced Gas Fee UI* which is an experimental feature in metamask, you can set the gas fee to *Aggresive* and that should work.
+
+![aggresive](.github/images/aggresive.gif)
+
+After making a few transactions, metamask should start automatically adjusting the gas fee to the optimal value.
+
+## Deploying the Contract
+
+We will use [remix](https://remix.ethereum.org/) to deploy the game contract on the Subnet.
+
+Go to the website and create a blank workspace.
+
+![workspace creation](.github/images/workspaceremix.gif)
+
+Next, create a folder and under that folder, create two files with the following contents:
+
+![new folder](.github/images/newfolder.png)
+
+### NativeMinterInterface.sol
+
+This is the `ContractNativeMinter` interface. We know the contract is deployed at `0x0200000000000000000000000000000000000001`.
+[See more about it here.](https://docs.avax.network/Subnets/customize-a-Subnet#minting-native-coins)
+
+```solidity
+// (c) 2022-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+interface NativeMinterInterface {
+    // Set [addr] to have the admin role over the minter list
+    function setAdmin(address addr) external;
+
+    // Set [addr] to be enabled on the minter list
+    function setEnabled(address addr) external;
+
+    // Set [addr] to have no role over the minter list
+    function setNone(address addr) external;
+
+    // Read the status of [addr]
+    function readAllowList(address addr) external view returns (uint256);
+
+    // Mint [amount] number of native coins and send to [addr]
+    function mintNativeCoin(address addr, uint256 amount) external;
+}
+```
+
+### Game.sol
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Headings should be surrounded by blank lines.

> MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]

[More info: MD022](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)